projects/WTI-API/src/main/controllers/ContestController.java

```diff
@@ -609,7 +640,17 @@ public Response getStandings(
 		}
 	}
 
-	
+	/**
```

Comment:
Is this debug code being left in production?

Reply:
No. in fact changes for #680 have already been pushed without debug 22, so this PR/branch may
be moot.